R/module_init_data.R

```diff
@@ -133,8 +127,13 @@ srv_init_data <- function(id, data, modules, filter = teal_slices()) {
       )
     })
 
-    # Adds signature protection to the datanames in the data
-    reactive(.add_signature_to_data(data_validated()))
+    reactive({
+      if (inherits(data_validated(), "teal_data")) {
+        .add_signature_to_data(data_validated())
+      } else {
+        data_validated() # error, qenv.error, shiny.silent.error, validate-error
```

Comment:
Not true anymore. It is `NULL` (initially if `teal_data` is unevaluated `teal_data_module`) or `teal_data`
```suggestion
        data_validated()
```